int64 direction
float32 step
---
std_msgs/Float32MultiArray stateVec
std_msgs/Float64MultiArray viz
bool done
int64 steps
float32 visibilityReward
